Frameworks/UIKit/UILabel.mm

```diff
@@ -301,19 +209,41 @@ - (void)_initUILabel {
 
     // on reference platform, default minimum font size is zero but always slightly bigger than zero in reality acccording to the
     // documentation
+    _adjustFontSize = NO;
     _minimumFontSize = 0.0001f;
     _minimumScaleFactor = 0.0;
     _useMinimumScaleFactor = NO;
     _numberOfLines = 1;
     [self setOpaque:FALSE];
 
-    _originalFontSize = [UIFont labelFontSize];
-    _font = [UIFont fontWithName:@"Segoe UI" size:_originalFontSize];
+    _font = [UIFont fontWithName:@"Segoe UI" size:[UIFont labelFontSize]];
 
     // TODO: Reevaluate whether or not this is the correct default mode for UILabels that are initialized via initWithFrame.
     //       Some of our test apps expect the initWithCoder path to default to UIViewContentModeScaleToFill (aka kCAGravityResize).
     [self setContentMode:UIViewContentModeRedraw];
-    [self _updateXamlElement];
+}
+
+- (void)_applyPropertesOnTextBlock {
+    [_textBlock setText:_text];
+    [_textBlock setTextAlignment:XamlUtilities::ConvertUITextAlignmentToWXTextAlignment(_alignment)];
+    XamlUtilities::ApplyLineBreakModeOnTextBlock(_textBlock, _lineBreakMode, self.numberOfLines);
+    [_textBlock setMaxLines:_numberOfLines];
+    UIColor* color = _textColor;
+    if (_isHighlighted && _highlightedTextColor != nil) {
+        color = _highlightedTextColor;
+    }
```

Comment:
nit: we have a few different paths for choosing the correct foreground text color; should we pull it out into a shared piece of code?